### ISSUE TITLE
Add id prefix to wandb .csv files

### ISSUE THREAD

**Issue by AUdaltsova** [openclimatefix/PVNet#267]
Currently validation .csvs save to wandb under the same name for all uns. We should add prefixes of id the same way it's dine for other artifacts.

**Comment by peterdudfield** [comment]
did this get sorted? @AUdaltsova

**Comment by AUdaltsova** [comment]
no they still save the same name. It's not a pressing problem I don't think as wandb does versioning and attributing to correct runs so you can see what is where, but should be done at some point as it's cumbersome right now

**Comment by chaitanya-rdY** [comment]
@AUdaltsova ,can I work on this issue??
if yes Please assign to me . so,I can start working..

**Comment by chaitanya-rdY** [comment]
@AUdaltsova , @peterdudfield , I am waiting for your reply.

**Comment by AUdaltsova** [comment]
Hi @chaitanya-rdY, thanks for your interest. This can be a bit tricky, do you mind confirming you know where to look to complete this before I assign you, maybe specifically what you'd do here?

**Comment by dfulu** [comment]
This is no longer needed after #428
